views/layout.html

```diff
@@ -123,6 +123,7 @@
 
   'close': __('Close'),
   'today': __('Today'),
+  'tomorrow': __('Tomorrow'),
```

Comment:
I couldn't find anywhere else we were using the string `'tomorrow`'. Did I miss it, or is this the first time we're using it?

If it is the first time, is this a good spot for it, and is there anything special I need to do to ensure it gets translated after the next release (if we're not already getting a translation from somewhere)?

Reply:
> I couldn't find anywhere else we were using the string 'tomorrow'. Did I miss it, or is this the first time we're using it?
Looking at the `lang/mailpoet.pot` file which holds all strings for translation, I can't find it, so apparently it's the first time.

> If it is the first time, is this a good spot for it
If this string is used only on the newsletters page, then perhaps it's more suitable to put in in `views/newsletters.html`. 

> is there anything special I need to do to ensure it gets translated after the next release (if we're not already getting a translation from somewhere)?
No special actions needed: if the normal release procedure is followed, the string will be pushed for translators automatically before a release and translated within a week or so. For small strings we usually don't require translations in advance.

Reply:
Thanks for the detailed response! You definitely answered my questions about the translation process.

> If this string is used only on the newsletters page, then perhaps it's more suitable to put in in `views/newsletters.html`.

I was torn between the two places and ended up adding it here because it felt appropriate to include it next to `today`, and it seems like a generic enough word that it might be nice to simply have it in the top level layout file, but it would be trivial to move it there later if need be, so I'll move it to `views/newsletters.html` for now if you think that's the best way to go.